src/app/demos/date-range-picker/date-range-picker-demo.component.html

```diff
@@ -0,0 +1,117 @@
+<div
+  class="date-range-demo"
+>
+  <form
+    id="screenshot-date-range-picker"
+    novalidate
+    [formGroup]="reactiveForm"
+  >
+    <div class="sky-form-group">
+      <sky-date-range-picker
+        formControlName="lastDonation"
+        label="Last donation"
+        [calculatorIds]="calculatorIds"
+        [dateFormat]="dateFormat"
+      >
+      </sky-date-range-picker>
+
+      <ng-container *ngIf="reactiveRange.errors && reactiveRange.touched">
+        <div *ngIf="reactiveRange.errors.skyDateRange?.errors.endDateBeforeStartDate"
+          class="sky-error-label"
+        >
+          End date must be before start date.
+        </div>
+        <div *ngIf="reactiveRange.errors.skyDate"
+          class="sky-error-label"
+        >
+          Invalid date entered.
+        </div>
+      </ng-container>
+    </div>
+  </form>
+
+  <h3>
+    Form control statuses
+  </h3>
+
+  <table>
+    <tr>
+      <th>Touched</th>
+      <td>{{ reactiveRange.touched }}</td>
+    </tr>
+    <tr>
+      <th>Pristine</th>
+      <td>{{ reactiveRange.pristine }}</td>
+    </tr>
+    <tr>
+      <th>Valid</th>
+      <td>{{ reactiveRange.valid }}</td>
+    </tr>
+  </table>
+
+  <h3>
+    Form control value
+  </h3>
+
+  <pre><code>{{ reactiveForm.value.lastDonation | json }}</code></pre>
+
+  <h3>
+    Manipulate value and state
+  </h3>
+
+  <button
+    class="sky-btn sky-btn-default"
+    type="button"
+    (click)="toggleDisabled()"
+  >
+    {{ reactiveForm.disabled ? 'Enable' : 'Disable'}} picker
+  </button>
+
+  <button
+    class="sky-btn sky-btn-default"
+    type="button"
+    (click)="resetForm()"
+  >
+    Reset form
+  </button>
+
+  <button
+    class="sky-btn sky-btn-default"
+    type="button"
+    (click)="setRange()"
+  >
+    Set specific range
+  </button>
+
+  <button
+    class="sky-btn sky-btn-default"
+    type="button"
+    (click)="setInvalidRange()"
+  >
+    Set invalid range
+  </button>
+
+  <button
+    class="sky-btn sky-btn-default"
+    type="button"
+    (click)="setInvalidDates()"
+  >
+    Set invalid dates
+  </button>
+
+  <button
+    class="sky-btn sky-btn-default"
+    type="button"
+    (click)="setCalculatorIds()"
+  >
+    Set custom calculators
+  </button>
+
+  <button
+    class="sky-btn sky-btn-default"
+    type="button"
+    (click)="setDateFormat()"
+  >
+    Set date format
```

Comment:
The `Set date format` button doesn't seem to visually change anything for the user. Am I missing something?

Reply:
Once you click the button, any new values that are selected by the datepicker will be formatted in a different way.

Reply:
Resolving since this seems to work as expected.